docs/extenders/SecureCredentials.md

```diff
@@ -0,0 +1,27 @@
+# Secure Credentials API Adoption
+
+Extenders may have a need for z/OS actions to be taken outside of Zowe Explorer using Zowe profiles that are managed by the Secure Credentials Store. Zowe Explorer API has a way for extenders to handle this situation.
+
+The `KeytarApi()` is available in `@zowe/zowe-explorer-api` version 1.15.1 and higher. Use of this API will need to be handled before the extension is registered with Zowe Explorer. This can be done by following the steps below:
```

Comment:
The first two sentences sound like prerequisites. Maybe, we should list them as pre-reqs? Then we can say something like: "Once you meet the pre-reqs, you can adopt/use the API.

Reply:
@IgorCATech no prereqs are needed.  This doc is to help Zowe Explorer extenders to understand how the new Zowe Explorer APi  `KeytarApi()` can be used/incorporated into their VS Code extension.